pipelines/rj_crm__disparo_template/flow.py

```
@@ -403,37 +403,39 @@ def rj_crm__disparo_template_sf(
+                )
+
+                if not partitions_path:
+                    raise ValueError("partitions_path is None - partition creation failed")
+
+                if not os.path.exists(partitions_path):
+                    raise ValueError(f"partitions_path does not exist: {partitions_path}")
+
+                print(f"Generated partitions_path: {partitions_path}")
+                if os.path.exists(partitions_path):
+                    files_in_path = []
+                    for root, dirs, files in os.walk(partitions_path):  # pylint: disable=unused-variable
```

Ruff (0.15.21)

[warning] Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)